### PR TITLE
Header

### PR DESCRIPTION
**Pull request by sabayounis**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
reviewing......

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+       
+
+    </div>
+    <aside>
+        <div class="promo-images">
+            <img src="assests/images/promo_iphone5s.jpg" alt="Iphone 5s">
+            <img src="assests/images/promo_iphone5c.jpg" alt="iphone 5c">
+            <img src="assests/images/promo_30_years.jpg" alt="promo">
+            <img src="assests/images/promo_supplier_responsibility.jpg" alt="supplier">
+        </div>
```
Comment (Oghenebrume50) [inline index.html:67]: These images are links so you should wrap them in an a tag and do well to make the links dead too

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,228 @@
+body{
+   margin: 0;
+   padding: 0;
+}
+.container{
+  margin-left: 42px;
+    margin-right: 100px;
+}
```
Comment (Oghenebrume50) [inline css/style.css:8]: These margins create spaces at both ends, is this needed?

**Review by Oghenebrume50 — COMMENTED** [review]
Hi @sabayounis this is a good start, a very good one, with some few more changes you would be done with this. 

currently, the display is a bit off see attached image 


- make stickler work for this repo 
- fix the outlined issues below

***Please do these things and request for a re-review :pray:***

Feel free to reach out to me for help, if you need it,  I got you, the whole TSE team is there to support you at all times :smile:  
[IMPORTANT] **You are not supposed to merge this pull request before it is reviewed**

File: css/style.css (hunk 2 above)
Comment (sabayounis) [reply css/style.css:8]: Oh okay thanks for the review let do it

**Pull request merged**